### PR TITLE
feat(signup): added helper text for address field

### PR DESCRIPTION
**Pull request by JacquesLarique**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-13601
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ (n/a)

## Description

Added a text helper for indian customer to insist on the fact they need to set the same address than the one on their documents.

## Related

- [ ] CMT-9473

### CODE REVIEW

File: packages/manager/apps/sign-up/src/form/component.js [hunk 1]
```diff
@@ -14,5 +14,7 @@ export default {
     isSmsConsentAvailable: '<',
     needkyc: '<',
     goToKycDocumentUploadPage: '<',
+    user: '<',
+    subsidiary: '<',
```
Comment (darsene) [inline packages/manager/apps/sign-up/src/form/component.js:18]: Why just not add `subsidiary` with the value `this.subsidiary || this.user.subsidiary`? You don't really need whole `user`

File: packages/manager/apps/sign-up/src/routing.js [hunk 2]
```diff
@@ -157,6 +159,8 @@ export const state = {
         customObject: { event: 'ACCOUNT_CREATION_VALIDATION' },
       });
     },
+
+    subsidiary: /* @ngInject */ ($location) => $location.search().ovhSubsidiary,
```
Comment (darsene) [inline packages/manager/apps/sign-up/src/routing.js:163]:
```suggestion
    subsidiary: /* @ngInject */ ($location, coreConfig) => $location.search().ovhSubsidiary || coreConfig.getUser().ovhSubsidiary,
```

File: packages/manager/apps/sign-up/src/routing.js [hunk 3]
```diff
@@ -57,6 +57,8 @@ export const state = {
       ssoAuthentication
         .getSsoAuthPendingPromise()
         .then(() => ssoAuthentication.user),
+
+    user: /* @ngInject */ (coreConfig) => coreConfig.getUser(),
```
Comment (darsene) [inline packages/manager/apps/sign-up/src/routing.js:61]:
```suggestion
```

File: packages/manager/modules/sign-up/src/form/details/details.scss [hunk 4]
```diff
@@ -51,4 +51,13 @@
...
     }
   }
+
+  .help-block {
+    display: block;
+    margin-top: 0;
+    font-size: 0.75rem;
+    font-weight: 400;
+    color: #000e9c;
+    margin-bottom: 10px;
```
Comment (anooparveti) [inline packages/manager/modules/sign-up/src/form/details/details.scss:61]: **nitpick:** Use utilities from bootstrap (margin, display, font-size, font-weight) & ui-kit (color).

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/sign-up/src/form/form.controller.js [hunk 5]
```diff
@@ -112,11 +110,10 @@ export default class SignUpFormAppCtrl {
...
-    if (ovhSubsidiary && ovhSubsidiary.match(/^[\w]{2}$/)) {
-      localStorage.setItem(OVH_SUBSIDIARY_ITEM_NAME, ovhSubsidiary);
+    if (this.subsidiary && this.subsidiary.match(/^[\w]{2}$/)) {
+      localStorage.setItem(OVH_SUBSIDIARY_ITEM_NAME, this.subsidiary);
     }
-    const subsidiary = ovhSubsidiary || this.user.ovhSubsidiary;
+    const subsidiary = this.subsidiary || this.user.ovhSubsidiary;
     this.subsidiaryLabelSuffix =
       SUBSIDIARIES_LABEL_SUFFIX[subsidiary] ||
       SUBSIDIARIES_LABEL_SUFFIX.DEFAULT;
```
Comment (darsene) [inline packages/manager/apps/sign-up/src/form/form.controller.js:119]:
```suggestion
     this.subsidiaryLabelSuffix =
      SUBSIDIARIES_LABEL_SUFFIX[this.subsidiary] ||
      SUBSIDIARIES_LABEL_SUFFIX.DEFAULT;
```

File: packages/manager/apps/sign-up/src/details/routing.js [hunk 6]
```diff
@@ -15,6 +17,8 @@ export const state = {
         name: `accountcreation::${field}::${value}`,
       });
     },
+    isIndianSubsidiary: /* @ngInject */ (subsidiary, user) =>
+      (subsidiary || user.ovhSubsidiary) === INDIAN_SUBSIDIARY,
```
Comment (darsene) [inline packages/manager/apps/sign-up/src/details/routing.js:21]:
```suggestion
    isIndianSubsidiary: /* @ngInject */ (subsidiary) =>
      subsidiary === INDIAN_SUBSIDIARY,
```

**Review by darsene — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**